gm4_pig_tractors/data/pig_tractors/functions/check_pig.mcfunction

```diff
@@ -0,0 +1,5 @@
+# @s = @a
+# Called from main
+tag @s remove gm4_on_pig

```

Comment:
```suggestion
tag @s remove gm4_on_pig
tag @s remove gm4_has_hoe
```

This will make sure players that left their pig holding a hoe get the tag removed.